docs/noise.ipynb

```diff
@@ -62,196 +62,673 @@
     "\n",
-    "A particular case of the asymmetric depolarizing channel is the case when each of the different Paulis occur with the same probability. This is\n",
-    "encapsulated in the `DepolarizingChannel` gate, which takes a probability `p`\n",
-    "such that each Pauli gate occurs with probability `p/3`.\n",
+    "An example of a channel which does not support the mixture protocol is the amplitude damping channel with parameter $\\gamma$ defined by Kraus operators\n",
     "\n",
-    "To construct channels, useful helpers are provided `cirq.asymmetric_depolarize`\n",
-    "and `cirq.depolarize`.\n",
+    "$$\n",
+    "M_0 = \\begin{bmatrix} 1 & 0  \\cr 0 & \\sqrt{1 - \\gamma} \\end{bmatrix} \n",
+    "\\text{and }\n",
+    "M_1 = \\begin{bmatrix} 0 & \\sqrt{\\gamma} \\cr 0 & 0 \\end{bmatrix} .\n",
+    "$$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "8e377332d2a7"
+   },
+   "outputs": [],
+   "source": [
+    "\"\"\"The amplitude damping channel is an example of a channel without a mixture.\"\"\"\n",
+    "channel = cirq.amplitude_damp(0.1)\n",
     "\n",
-    "Another common case is when only a Pauli X (bit flip) can occur, or\n",
-    "when only a Pauli Z (phase flip) can occur. These correspond to\n",
-    "`BitFlipChannel` and `PhaseFlipChannel` with helpers `cirq.bit_flip` and\n",
-    "`cirq.phase_flip`."
+    "if cirq.has_mixture(channel):\n",
+    "    print(f\"Channel {channel} has a _mixture_ or _unitary_ method.\")\n",
+    "else:\n",
+    "    print(f\"Channel {channel} does not have a _mixture_ or _unitary_ method.\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {
-    "id": "ebed67376062"
+    "id": "6f3ae4337219"
    },
    "source": [
-    "### GeneralizedAmplitudeDampingChannel and AmplitudeDampingChannel\n",
-    "\n",
-    "The generalized amplitude damping channel models the effect of energy\n",
-    "dissipation to a surrounding environment as well as dephasing that\n",
-    "does not exchange energy. The amplitude damping channel only models dissipation of energy to a surrounding environment. \n",
+    "To summarize:\n",
+    "\n",
+    "* Every channel supports the `cirq.channel` protocol. (Note this includes gates like `cirq.X` and `cirq.CNOT`.)\n",
+    "  - If magic method `_channel_` is not defined, `cirq.channel` looks for `_mixture_` then for `_unitary_`.\n",
+    "* A subset of channels which support `cirq.channel` also support the `cirq.mixture` protocol.\n",
+    "  - If magic method `_mixture_` is not defined, `cirq.mixture` looks for `_unitary_`.\n",
+    "* A subset of channels which support `cirq.mixture` also support the `cirq.unitary` protocol.\n",
+    "  - If `_unitary_` is not defined for an object, the object is probably not meant for a circuit!"
```

Comment:
Doubly confused now, because we use an amplitude_damping channel below in a circuit. So now I'm not really sure what this sentence means.

Reply:
The inclusions should be reversed - something in a circuit must have a channel, not necessarily a mixture or a unitary, though. Some channels have mixtures (e.g. cirq.bit_flip), and some channels have unitaries (e.g. cirq.X). 

So, {channels with unitaries} ⊂ {channels with mixtures} ⊂ {channels}.

Thanks for pointing this out, I'm sure others who read it will have the same confusion. I'll add an example to hopefully clarify.

Reply:
Alright, perhaps remove the last line here though?  "If `_unitary_` is not defined for an object, the object is probably not meant for a circuit!" sounds like I can't add objects that aren't unitary to a circuit, which is confusing.